getting-started/events.md

```diff
@@ -0,0 +1,268 @@
+
+
+### Common event fields
+
+All events have some common fields, we will describe them here to avoid duplication:
+
+| name        | puprose                                        | filled by | type   |
+|-------------|------------------------------------------------|-----------|--------|
+| `id`        | id of the event                                | server    | string |
+| `custom_id` | is an ID that you can set for your own purpose | client    | string |
+| `order`     | event with lower `order` are hronologicaly before | server | int    |
+| `author_id` | this is id of sender of the event              | server    | string |
+| `text`      | this is payload that you can send as event (in example message content) | client | string |
+| `recipients` | posibble values are `"all"`(default) and `"agents"` | client | string |
+| `properties` | event properties, more about this in future properties section | client | properties object |
+
+
+### Message
+
+Message is the most common and self-explanatory event type. It allows sending some text to other chat users. The data format looks like this:
+
+```js
+{
+  "id": "0affb00a-82d6-4e07-ae61-56ba5c36f743",  // generated by server side
+  "custom_id": "12345-bhdsa",
+  "order": 1,  // generated by server side
+  "type": "message",
+  "author_id": "b7eff798-f8df-4364-8059-649c35c9ed0c",
+  "timestamp": 1473433500,  // generated by server side
+  "text": "hello there",
+  "recipients": "all",
+  "properties": {
+    // "Properties" object
+  }
+}
+```
+
+
+### Annotation
+
+Adds annotation to last thread. It is important to mention that sending annotation cannot start a new thread (even if there is no active thread in a chat), it always goes to latest thread that alredy exist.
+
+```js
+{
+  "id": "0affb00a-82d6-4e07-ae61-56ba5c36f743", // generated by server side
+  "custom_id": "12312.301231238591134",
+  "order": 1, // generated by server side
+  "type": "annotation",
+  "author_id": "b7eff798-f8df-4364-8059-649c35c9ed0c",
+  "timestamp": 1473433500, // generated by server side
+  "text": "Example annotation",
+  "annotation_type": "rating",
+  "properties": {
+    // "Properties" object
+  }
+}
+```
+
+`annotation_type` - type of the annotation. This field cannot be empty and is customizable, you can have your own annotation types (i.e. we use "rating" type)
+
+
+### Filled form
+
+Filled form is event containing some data from form. Here's what it looks like:
+
+```js
+{
+  "id": "0affb00a-82d6-4e07-ae61-56ba5c36f743", // generated by server side
+  "custom_id": "12312.301231238591134",
+  "order": 4, // generated by server side
+  "type": "filled_form",
+  "author_id": "b7eff798-f8df-4364-8059-649c35c9ed0c",
+  "timestamp": 1473433500,  // generated by server side
+  "properties": {
+    // "Properties" object
+  }
+  "fields": [
+    // "Field" objects
+  ]
+}
+```
+
+There are 5 different types of field from filled form, all of these types can be mixed together in one filled form event.
```

Comment:
There are 5 different types of field from filled form, all of these types can be mixed together in one filled form event. - > There are 5 different field types. All of them can be used in a single filled form event.

Reply:
chodziło mi tutaj o to, że można używać różnych typów na raz, w sensie, że do tablicy "fields" można wrzucić jednocześnie np text i email